state-chain/pallets/cf-broadcast/src/lib.rs

```diff
@@ -476,72 +435,16 @@ pub mod pallet {
 		///
 		/// ## Errors
 		///
-		/// - [InvalidBroadcastAttemptId](Error::InvalidBroadcastAttemptId)
-		/// - [InvalidSigner](Error::InvalidSigner)
+		/// - [InvalidBroadcastId](Error::InvalidBroadcastId)
 		#[pallet::call_index(0)]
-		#[pallet::weight(T::WeightInfo::transaction_signing_failure())]
-		pub fn transaction_signing_failure(
+		#[pallet::weight(T::WeightInfo::transaction_failed())]
```

Comment:
I would prefer if we remained a bit more defensive here: we can keep the old extrinsic (but have it just return an Error), and add a new one with the new functionality.

Will change.

Reply:
(4b62708001058a40fd1bee4d1bc2cb3af5daa42f)